package.json

```diff
@@ -53,10 +53,10 @@
     "nuclear-js": "1.4.0",
     "postis": "2.2.0",
     "prop-types": "15.6.0",
-    "react": "15.6.1",
-    "react-dom": "15.6.1",
+    "react": "16.0.0",
+    "react-dom": "16.0.0",
     "react-i18next": "4.8.0",
-    "react-native": "0.42.3",
+    "react-native": "0.49.3",
```

Comment:
can we pin this to a commit which includes the fix for the FIXME above instead?

Reply:
which FIXME ?

Reply:
https://github.com/jitsi/jitsi-meet/pull/2105/files#diff-a9d0201e8c5bdc946e54218bb9206174R26

Reply:
do you know how to pin to a commit ? (I will google it otherwise, but just thought you may now)

Reply:
this commit is contained in RN v0.50.0-rc.2 but then we'd be on not stable. I'm not sure if it's worth the effort - it's just a flow ignore

Reply:
Ack